swingset-extensions/load-sample-mdx/index.ts

```diff
@@ -0,0 +1,36 @@
+import * as fs from 'fs'
+import * as path from 'path'
+
+import { serialize } from 'next-mdx-remote/serialize'
+import { anchorLinks } from '@hashicorp/platform-remark-plugins'
+import jumpToSection from '@hashicorp/platform-remark-plugins/plugins/jump-to-section'
+
+import { GetStaticPropsContext } from 'next'
+
+const loadSampleMdx = async (ctx: GetStaticPropsContext, staticProps) => {
```

Comment:
Since this is pretty tightly scoped 💭 
```suggestion
const loadSampleMdxForDocsPage = async (ctx: GetStaticPropsContext, staticProps) => {
```

Reply:
I'll update the file name too to match this change